webpack-hot-dev-server.config.js

```diff
@@ -62,7 +62,7 @@ module.exports = require("./make-webpack-config")(rules, {
     chunkFilename: "[id].js"
   },
   devServer: {
-    port: 3200,
+    port: 8082,
```

Comment:
We can probably not include this in `master` we could maybe have it set as a environment variable. (Or use environment variable if its set, and if not default to a port number.)